src/gui/src/UI/UIAlert.js

```diff
@@ -19,31 +19,72 @@
 
 import UIWindow from './UIWindow.js'
 
-function UIAlert(options){
+function UIAlert(options) {
     // set sensible defaults
-    if(arguments.length > 0){
+    if (arguments.length > 0) {
         // if first argument is a string, then assume it is the message
-        if(window.isString(arguments[0])){
+        if (window.isString(arguments[0])) {
             options = {};
             options.message = arguments[0];
         }
         // if second argument is an array, then assume it is the buttons
-        if(arguments[1] && Array.isArray(arguments[1])){
+        if (arguments[1] && Array.isArray(arguments[1])) {
             options.buttons = arguments[1];
         }
     }
 
     return new Promise(async (resolve) => {
         // provide an 'OK' button if no buttons are provided
-        if(!options.buttons || options.buttons.length === 0){
+        if (!options.buttons || options.buttons.length === 0) {
             options.buttons = [
-                {label: i18n('ok'), value: true, type: 'primary'}
+                { label: i18n('ok'), value: true, type: 'primary' }
             ]
         }
+        // Define alert types
+        const alertTypes = {
+            error: { icon: "danger.svg", title: "Error!", color: "#D32F2F" },
+            warning: { icon: "warning-sign.svg", title: "Warning!", color: "#FFA000" },
+            info: { icon: "reminder.svg", title: "Info", color: "#1976D2" },
+            success: { icon: "c-check.svg", title: "Success!", color: "#388E3C" },
+            confirm: { icon: "question.svg", title: "Are you sure?", color: "#555555" }
+        };
 
-        // set body icon
-        options.body_icon = options.body_icon ?? window.icons['warning-sign.svg'];
-        if(options.type === 'success')
+        // Set default values
+        const alertType = alertTypes[options.iconType] || alertTypes.warning;
+        options.message = options.title || alertType.title;
+        options.body_icon = options.body_icon ?? window.icons[alertType.icon];
+        options.color = alertType.color;
```

Comment:
Maybe we should allow an override here.

```suggestion
        options.color = options.color ?? alertType.color;
```